model/riscv_insts_zilsd.sail

```diff
@@ -0,0 +1,39 @@
+/*=======================================================================================*/
+/*  This Sail RISC-V architecture model, comprising all files and                        */
+/*  directories except where otherwise noted is subject the BSD                          */
+/*  two-clause license in the LICENSE file.                                              */
+/*                                                                                       */
+/*  SPDX-License-Identifier: BSD-2-Clause                                                */
+/*=======================================================================================*/
+
+function clause extensionEnabled(Ext_Zilsd) = true & xlen == 32
+
+/* ****************************************************************** */
+union clause ast = LOAD_RV32 : (bits(12), regidx, regidx)
```

Comment:
`LOAD_DOUBLE` might be a better name for the AST constructor than `LOAD_RV32`.   It's unfortunate that the spec re-uses `LOAD`.  (Could we ask the spec authors to change this before ratification?) Similarly for `STORE_RV32`.